stock_quant_internal_transfer/models/stock_quant.py

```diff
@@ -0,0 +1,14 @@
+## -*- coding: utf-8 -*-
+# Copyright 2017 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class StockQuant(models.Model):
+    _inherit = 'stock.quant'
+
+    picking_id = fields.Many2one(
```

Comment:
@mra-odoo May I know why this field is needed?

Reply:
@TimLai125 This field is for history purpose and filtering purpose so when you are on quant view you can get idea this quant has been created from this picking...